expert-forecast/expert-forecast-app/app.R

```diff
@@ -10,18 +10,18 @@ library(googlesheets4)
 
```

Comment:
in general it isn't a good idea to call the tidyverse package for code other people are going to need to use. It makes it hard to know what the dependencies actually are.